### DISCUSSION TITLE
Ideas for add-ons or tweaks for bedrock edition

### DISCUSSION THREAD

**Discussion by buoyant-ship** [BedrockTweaks/Files#435] [Ideas]
Dear Drav.dev Team,

I hope you are doing well. I suggest an add-on for Minecraft Bedrock Edition to enhance players' gameplay. Specifically, I propose an add-on allowing Bedrock players to use their off-hands for tools and items such as fireworks, torches, food, and to place blocks. This feature would mirror the functionality available to Java Edition players and improve accessibility and convenience for Bedrock users.

Introducing this add-on could offer players a smoother and more enjoyable gaming experience and attract players who prefer the Java Edition's convenience, capabilities, and general quality of life. This potential increase in the player base with your add-ons and tweaks could be a significant commercial benefit for your cause toward a better Bedrock Edition.

The community would welcome this addition, which would significantly contribute to the Bedrock Edition's ongoing development. Using quality-of-life mods, resource packs, and add-ons would be a massive step towards making the game more enjoyable for all players.

Thank you for considering my suggestion. I am eager to hear your thoughts and look forward to the possibility of further discussing this idea with you.

Best regards, Brett Green (Buoyant)
way of contact: [brettcongreen@gmail.com](mailto:brettcongreen@gmail.com) Discord: bretgreen

**Comment by SquatchHunter** [comment]
unfortunately, I do not believe this is possible as it just breaks the game entirely thanks to hard coded behaviours we simply cannot acces
